### PR TITLE
Fix(DB/Creature): Set correct faction flag for Aggi Rumplestomp.

### PR DESCRIPTION
**Pull request by temperrr**
## Changes Proposed:
-  Changes faction flag to be the same as the other creatures around him.
-  

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/3403

## Tests Performed:

- Not tested
- 

## How to Test the Changes:

1. Go to Coldtooth mine in AV and see if Aggi has no longer the Alliance Pvp symbol on his name and can be properly killed.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by fputs** [comment]
Aggi is set to the correct faction and does not have the PvP symbol on his nameplate. Looks good!

2022-04-26-080604_2560x1440_scrot

File: data/sql/updates/pending_db_world/rev_1650793025980507979.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1650793025980507979');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1650793025980507979.sql:1]: delete this

Comment (temperrr) [reply data/sql/updates/pending_db_world/rev_1650793025980507979.sql:1]: Why

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1650793025980507979.sql:1]: Because it doesnt exist anymore

**Review by Nefertumm — APPROVED** [review]

**Pull request merged**